_data/simple-icons.json

```diff
@@ -2300,6 +2300,11 @@
             "hex": "6FDA44",
             "source": "https://www.upwork.com/press/"
         },
+	{
+		"title": "Unity",
+		"hex": "000000",
+		"source": "https://vectors.pro/unity-3d-vector-icon-logo/"
```

Comment:
We'd prefer an official source if we can get one. In this case https://unity3d.com/ actually includes [an SVG](https://unity3d.com/profiles/unity3d/themes/unity/images/ui/ui/unity-logo-white.svg) in their navbar - all that's left to do is crop, resize and annotate it.

Reply:
Thank you for your kind anwer! I've modified all thing you mentioned directly. :D!